axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/StandardIndexManager.java

```diff
@@ -68,6 +69,7 @@
     private final MeterFactory.RateMeter indexCloseMeter;
     private final RemoteAggregateSequenceNumberResolver remoteIndexManager;
     private ScheduledFuture<?> cleanupTask;
+    private final AtomicLong maxMmapIndex = new AtomicLong();
```

Comment:
the name of this field is not clear to me: I suggest in alternative
- minMmapIndex (the segment should be greater than this index to use Memory Mapped Files)
- useMmapAfterIndex
- maxChannelIndex (the segment shoudl be less than this index to use FileChannel)
- useChannelBeforeIndex

or, if you want to be generic:
- mMapThreshold

updateMaxMmapIndex method name should be consisten with the field name.